lib/filter-headers.js

```diff
@@ -1,23 +1,41 @@
 'use strict';
-const ACCEPT_HEADERS = ['accept-language', 'referer', 'user-agent'];
-/**
- * Filter the request headers that are passed to fragment request.
- * @callback filterHeaders
- *
- * @param {Object} attributes - Attributes object of the fragment node
- * @param {string} attributes.public - Denotes the public fragment.
- * @param {Object} request - HTTP Request object
- * @param {Object} request.headers - request header object
- * @returns {Object} New filtered header object
- */
-module.exports = (attributes, request) => {
-    const { public: publicFragment } = attributes;
-    const { headers = {} } = request;
-    // Headers are not forwarded to public fragment for security reasons
+const ACCEPT_REQ_HEADERS = ['accept-language', 'referer', 'user-agent'];
+const ACCEPT_RES_HEADERS = ['location', 'set-cookie'];
```

Comment:
I also realised later that not passing Location for public fragments breaks expected behaviour on a 30x.
Considering this and the fact the function signature will be different I propose to not include a default filter.